pkg/rtc/clientinfo.go

```diff
@@ -85,7 +89,7 @@ func (c ClientInfo) SupportsChangeRTPSenderEncodingActive() bool {
 }
 
 func (c ClientInfo) ComplyWithCodecOrderInSDPAnswer() bool {
-	return !(c.isLinux() && c.isFirefox())
+	return !((c.isLinux() || c.isAndroid()) && c.isFirefox())
```

Comment:
Is Firefox on all Linux busted including desktop?

Reply:
I have not tested it yet, but when I tested the firefox again, found the ua changed from linux.firefox(yesterday) to android.firefox modile(today).

Reply:
oh boy, fun with user agents 🙁